nf_core/modules/test_yml_builder.py

```diff
@@ -330,13 +334,12 @@ def run_tests_workflow(self, command):
                     "It looks like Nextflow is not installed. It is required for most nf-core functions."
                 )
         except subprocess.CalledProcessError as e:
-            output = rich.markup.escape(e.output.decode())
-            raise UserWarning(f"Error running test workflow (exit code {e.returncode})\n[red]{output}")
+            raise UserWarning(f"Error running test workflow (exit code {e.returncode})\n[red]{e.output.decode()}")
```

Comment:
How come you've removed the escaping stuff? I added that quite recently to solve an issue that people were hitting where the Nextflow output had square brackets in it..

Reply:
There was a collision and I left it as it was when I started the PR, sorry! I guess line 342 below needs to be fixed also, right?

Reply:
Ah ok - yeah just copy + paste the deleted lines from the PR diff over the added lines. Hopefully the PR diff will then show no changes.

Reply:
```suggestion
            output = rich.markup.escape(e.output.decode())
            raise UserWarning(f"Error running test workflow (exit code {e.returncode})\n[red]{output}")
```